static/js/pathfinder.js

```diff
@@ -180,8 +208,17 @@ function loadGraph(element, address){
     restRequest('GET', null, display, address);
 }
 
+function renameVulnerabilityReport(){
+    current_report = $('#altVulnerabilityReport').val();
+    let new_name = $('#newReportName').val();
+    console.log(current_report)
+    console.log(new_name)
+    apiV2('PATCH', '/plugin/pathfinder/api', {'index':'report','id':current_report, 'rename':new_name});
+    reloadReports();
+}
+
 function downloadVulnerabilityReport(){
-    current_report = $('#vulnerabilityReport').val();
+    current_report = $('#altVulnerabilityReport').val();
```

Comment:
What is denoted by the variable having the `alt` prefix?

Reply:
The `reports` tab was removed, as is caused feature duplication and created an issue with the jQuery dropdown of vulnerability reports.  This id was removed.  Thanks for the input, I'm much happier with keeping all the CRUD operations in one place.